projects/components/src/list-view/list-view.component.ts

```diff
@@ -18,6 +23,9 @@ import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
   `
 })
 export class ListViewComponent {
+  @Input()
+  public headerKeys?: [string, string];
```

Comment:
ListViewRecord also has `key` and `value` properties. So, instead of `headerKeys` should we have two properties: `keyHeaderLabel: string` and 'valueHeaderLabel: string`

Reply:
Yeah we can have that, But instead of making these two different inputs, we can just create and interface  with these two attributes and use that.

Reply:
Yeah, like 'headerLabel?: ListHeaderLabel;'

Interface ListHeaderLabel { key: string; value: string;}

Reply:
I guess,` interface ListHeader { keyLabel: string; valueLabel: string}` this would be better?

Reply:
Yeah. This is fine too!